src/custom/state/operator/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { createAction } from '@reduxjs/toolkit'
+
+// modeled after state/user/actions
+export interface SerializedToken {
+  chainId: number
+  address: string
+  decimals: number
+  symbol?: string
+  name?: string
+}
+
+export interface Tip {
+  value: string
+  expiration: string // serialazable date or time
+}
+
+export const updateTip = createAction<{ token: SerializedToken; tip: Tip }>('operator/updateTip')
```

Comment:
i understand this is just a temp thing, but would we need the entire serialised token in state or do u guys think `tokenId` being `token.address` be fine? @anxolin @Velenir

Reply:
likely no
address would probably suffice
and a special flag for ETH

Reply:
> address would probably suffice

👌

> and a special flag for ETH? Or we consider WETH tip for ETH

We will never ask for the tip of Ether, cause the tip refers only to the sell token, and it's not possible to sell Ether.